### PR TITLE
Testw chris testing

### PR DESCRIPTION
**Pull request by StephenGitHuber**
Hey Chris, latest and greatest so far!

### CODE REVIEW

File: TestwChrisV2.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: RealTestingwChris ;)
+menu_order: 4
+post_status: draft
+post_excerpt: See more on testing time on GitHub and following updated instructions
```
Comment (SienciLabs) [inline TestwChrisV2.md:5]: Hey I don't like this

File: TestwChrisV2.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+## Subheader
+
+This is the place for **Bold** and *Italics*
+
+Link to me [HERE](https://resources.sienci.com/view/slb-manual/#wiring)
+
+<https://resources.sienci.com/view/slb-manual/#wiring>
+
```
Comment (SienciLabs) [inline TestwChrisV2.md:42]: THis should be updated

**Review by SienciLabs — CHANGES_REQUESTED** [review]
This current version of your branch could use some more work

**Comment by StephenGitHuber** [comment]
Fixed and ready for review again

**Pull request closed without merging**